fdbserver/include/fdbserver/EncryptionUtil.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * EncryptionUtil.h
```

Comment:
nit: There exists a `flow/EncryptUtils.cpp`, can we please rename this file to `EncryptionOpsUtils.cpp` and rename the method to `isEncryptionOpSupported`?